virttest/qemu_vm.py

```diff
@@ -4209,17 +4224,10 @@ def get_block_old(self, blocks_info, p_dict={}):
                     return block.split(":")[0]
         else:
```

Comment:
This `else` clause handles QMP output, so we can add comment to help understand it.

Reply:
updated thanks.